### PR TITLE
Added "albumId" to "getPhotos" options

### PR DESCRIPTION
**Pull request by z1c0**
The "options" object passed to "getPhotos" now looks for an "albumId" property and retrieves only photos for this specific album.

Updated sample code.

### CODE REVIEW

**Comment by esteban-uo** [comment]
@z1c0 thanks!, your commit is behind master and is causing the conflict with your previous PR, could you please update it?

File: src/picasa.js [hunk 1]
```diff
@@ -112,10 +112,13 @@ function getPhotos (accessToken, options, callback) {
 
   if (options.maxResults) accessTokenParams['max-results'] = options.maxResults
```
Comment (esteban-uo) [inline src/picasa.js:114]: nice option!, can we apply it in this way?

``` js
const albumPart = options.albumId ? `/albumid/${options.albumId}` : '';
```

**Review by esteban-uo — CHANGES_REQUESTED** [review]
Just need to update a small style code suggestion & merge conflict

**Comment by hbakhtiyor** [comment]
@esteban-uo why not merged?

**Comment by esteban-uo** [comment]
@hbakhtiyor I missed the notification, thanks for the reminder. And sorry @z1c0 and thanks!

**Review by esteban-uo — COMMENTED** [review]
Approved. But seems to be that @z1c0 fork was deleted and I can't merge the PR. I'm going to add it by myself next week.

**Comment by esteban-uo** [comment]
@z1c0 merged, thanks for your PR and thx @hbakhtiyor for the reminder

**Pull request closed without merging**